staging/src/k8s.io/apiserver/pkg/endpoints/watch_test.go

```diff
@@ -272,7 +272,7 @@ func TestWatchRead(t *testing.T) {
 		// TODO: yaml stream serialization requires that RawExtension.MarshalJSON
 		// be able to understand nested encoding (since yaml calls json.Marshal
 		// rather than yaml.Marshal, which results in the raw bytes being in yaml).
-		// Same problem as thirdparty object.
+		// Same problem as customresource object.
```

Comment:
I got this as well in another PR, didn't commit it, continued working and suddenly it was gone. I fear this is something deep in the dockerized rsync build logic, somewhere between dragons, bash code and make fules.